Cargo.toml

```diff
@@ -13,7 +13,7 @@ crate-type = ["cdylib", "rlib"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-near-sdk = "5.0.0-alpha.2"
+near-sdk = { version = "5.0.0-alpha.2", default-features = false, features = ["unit-testing"]}
```

Comment:
I really hope we can make the defaults really great, so we don't need to expect developers to tweak it unless they explicitly need some specific feature. This is a bad UX when you need to learn tricks to get something to work productively.